### PR TITLE
feat: add language files

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: semantics.json [hunk 1]
```diff
@@ -112,7 +112,39 @@
     "label": "Localization",
     "name": "l10n",
     "type": "group",
-    "fields": []
+    "common": true,
```
Comment (boyum) [inline semantics.json:115]: Was `common` required for it to work?

File: src/components/Settings/Settings.tsx [hunk 2]
```diff
@@ -17,13 +18,14 @@ export const Settings: React.FC<SettingsProps> = ({
   handleAnswerModeChange,
   handleLanguageModeChange,
 }) => {
-  const answerModeText = 'Change answer mode'; // TODO: Translate
-  const languageModeText = 'Change language'; // TODO: Translate
+  const { t } = useTranslation();
+
+  const answerModeText = t('answerModeLabel');
+  const languageModeText = t('languageModeLabel');
```
Comment (boyum) [inline src/components/Settings/Settings.tsx:24]: We could move these calls into the component render if that's cleaner. (`<p>{t('...')}</p>`)

File: src/h5p-vocabulary-drill.tsx [hunk 3]
```diff
@@ -29,21 +29,24 @@ class VocabularyDrillContentType
 
     // TODO: Translate
     const title = this.extras?.metadata.title ?? 'Vocabulary drill';
```
Comment (boyum) [inline src/h5p-vocabulary-drill.tsx:31]: Should we add a default title in semantics, or just assume that it's always set?

```suggestion
    const title = this.extras?.metadata.title ?? '';
```

File: language/nb.json [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    },
+    {
+      "label": "Lokalisering",
+      "fields": [
+        {
+          "label": "Ingen gyldige ord",
+          "default": "Ingen gyldige ord funnet. Kontroller ordene dine og prøv igjen."
+        },
+        {
+          "label": "Åpne innstillinger aria tekst",
```
Comment (boyum) [inline language/nb.json:77]: I don't think `aria` is anything users will know about.
```suggestion
          "label": "Åpne innstillinger (vises kun for skjermlesere)",
```

File: language/nb.json [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+        {
+          "label": "Skill mellom store og små bokstaver",
+          "description": "Sørger for at svaret til brukeren må være nøyaktig det samme som i oppgaven."
+        },
+        {
+          "label": "Krev at alle felter er besvart før fasit gis"
+        },
+        {
+          "label": "Godta mindre stavefeil",
+          "description": "Hvis aktivert, vil et svar også telle som riktig med mindre stavefeil (3-9 tegn: 1 stavefeil, mer enn 9 tegn: 2 stavefeil). Dette alternativet er bare gyldig hvis svarmodus er satt til Fyll ut"
```
Comment (boyum) [inline language/nb.json:116]:
```suggestion
          "description": "Hvis denne er aktivert vil et svar også telle som riktig med mindre stavefeil (3-9 tegn: 1 stavefeil, mer enn 9 tegn: 2 stavefeil). Dette alternativet er bare gyldig hvis svarmodus er satt til Fyll ut"
```

File: language/nb.json [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+        },
+        {
+          "label": "Randomiser"
```
Comment (boyum) [inline language/nb.json:119]:
```suggestion
          "label": "Velg tilfeldige ord"
```

File: semantics.json (hunk 1 above)
Comment (henriettemoe) [reply semantics.json:115]: If we don't use `common` the translations will not be placed under the collapse "Text overrides and translations" but a collapse looking like this:
Skjermbilde 2023-02-21 kl  10 48 14

Comment (henriettemoe) [reply semantics.json:115]: But it does seems to work without `common`

File: language/nn.json [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+        },
+        {
+          "label": "Krev at alle felt er besvart før fasit gis"
+        },
+        {
+          "label": "Godta mindre stavefeil",
+          "description": "Viss aktivert, vil eit svar også telja som rett med mindre stavefeil (3-9 teikn: 1 stavefeil, meir enn 9 teikn: 2 stavefeil). Dette alternativet er berre gyldig viss svarmodus er sett til Fyll ut"
+        },
+        {
+          "label": "Randomiser"
```
Comment (boyum) [inline language/nn.json:119]:
```suggestion
          "label": "Vel tilfeldige ord"
```

File: semantics.json (hunk 1 above)
Comment (boyum) [reply semantics.json:115]: Ahhh common it is then!

File: src/h5p-vocabulary-drill.tsx (hunk 3 above)
Comment (henriettemoe) [reply src/h5p-vocabulary-drill.tsx:31]: Doesn't seem possible to save the content without any title. Don't know if there are any loopholes

Comment (boyum) [reply src/h5p-vocabulary-drill.tsx:31]: Then let's just remove the fallback title

**Review by boyum — APPROVED** [review]

**Pull request merged**